### PR TITLE
Add installation and verification instructions

### PR DESCRIPTION
**Pull request by calebhearth**

### CODE REVIEW

File: dist/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The best way to verify the release is to examine the GPG signature. You can find
+my public key at http://calebthompson.io/pubkey.asc,
+<https://pgp.mit.edu/pks/lookup?op=get&search=0x1621ADC2A0ACE70A>, and via GnuPG
+with `gpg --recv-keys A0ACE70A`.
+
+You can find a signed and verifiable statement of my ownership of that key at
+<https://github.com/calebthompson/i-am>.
+
+Once you have that key, verification is pretty easy:
```
Comment (derekprior) [inline dist/README.md]: Once you have that key, verify by:

File: dist/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Pick the binary that matches your operating system and architecture. For
+example, on my Macbook Pro I would use `ftree-darwin-amd64`. Copy that somewhere
+in your path and rename to `ftree`. Presto bango.
+
+## Verifying
+
+The best way to verify the release is to examine the GPG signature. You can find
+my public key at http://calebthompson.io/pubkey.asc,
+<https://pgp.mit.edu/pks/lookup?op=get&search=0x1621ADC2A0ACE70A>, and via GnuPG
+with `gpg --recv-keys A0ACE70A`.
```
Comment (derekprior) [inline dist/README.md]: How is this more secure than verifying with the sha? Presumably if I access the git repo to edit the sha, I can also edit the git repo to edit this URL or key id?

Comment (calebhearth) [inline dist/README.md]: You can get pseudo-trust by verifying the calebthompson/i-am repo. It's good enough to establish that the person with @calebthompson is the person who committed these files.

Comment (calebhearth) [inline dist/README.md]: And if you happen to be within 3 hops of me in the WoT you have for real (PGP) trust as well.

Comment (calebhearth) [inline dist/README.md]: Further, a SHA sum would be easier to fake as the signature is shorter. Here's the signature on `ftree-darwin-amd64`:

```
-----BEGIN PGP SIGNATURE-----

iQEcBAABCgAGBQJWFrX8AAoJEBYhrcKgrOcKkvgH/3hMp0JywCLSFLfPMIF0Cca8
WvpMcKsR3WY4s5iUm1djdb8Bp2eLxhFzWe78JM+KxAafuc500CjbLXdoF/LISDM7
Be2lFjdOdDwpMD5UcJDoj4wUE2wnHKAdPR36zlNhLNV+qSbnbb16khY6ZFTeXPZE
f2Cml3GxcCHag6bKKjy1GTZacDiBKTeOgf1lkFmqyccWAPdCO+5qeLs2dGbubt36
v3KU8JMOjIXFVbEJi+P09vuv+NWT9QJvnp5ZepQlGqLoWvMynUpMO7Y84u4+4Ca9
b05/W+WkFGcwWvZGiyBcXzAGUMyiKuvtsjGW/XOz/MB5igfhIJqQX2nN5lvYoJQ=
=IcUU
-----END PGP SIGNATURE-----
```

And the SHA sum:

```
a7342c0f695e229092aa304e2ffd7908035e9ff6e237acdf4650e2783c862212  ftree-darwin-amd64
```

File: dist/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Once you have that key, verification is pretty easy:
+
+```shell
+$ gpg --verify ftree-<os>-<arch>.sig ftree-<os>-<arch>
+```
+
+If you don't have GnuPG, you can verify the SHA sums.
+
+```shell
+$ shasum --algorithm 256 --check ftree-<os>-<arch>.sha256
```
Comment (calebhearth) [inline dist/README.md:26]: Another thing I can do here is store all checksums together and they can be verified like:

```
❯ shasum -a 256 --check checksums 
ftree-darwin-386: OK
ftree-darwin-amd64: OK
ftree-freebsd-386: OK
ftree-freebsd-amd64: OK
ftree-freebsd-arm: OK
ftree-linux-386: OK
ftree-linux-amd64: OK
ftree-linux-arm: OK
ftree-openbsd-386: OK
ftree-openbsd-amd64: OK
ftree-windows-386: OK
ftree-windows-amd64: OK
```

**Comment by calebhearth** [comment]
@derekprior thanks
@mike-burns any thoughts?

File: dist/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Installing
+
+Pick the binary that matches your operating system and architecture. For
+example, on my Macbook Pro I would use `ftree-darwin-amd64`. Copy that somewhere
+in your path and rename to `ftree`. Presto bango.
+
+## Verifying
+
+The best way to verify the release is to examine the GPG signature. You can find
```
Comment (mike-burns) [inline dist/README.md]: I believe this is a PGP signature, though you can use GPG to verify it.

**Comment by mike-burns** [comment]
Works for me; I just have one pedantic wording complaint.

**Pull request merged**